selfdrive/car/hyundai/values.py

```diff
@@ -202,21 +202,31 @@ class Buttons:
     (Ecu.fwdRadar, 0x7d0, None): [
       b'\xf1\x00LX2_ SCC FHCUP      1.00 1.04 99110-S8100         \xf1\xa01.04',
       b'\xf1\x00LX2 SCC FHCUP      1.00 1.04 99110-S8100         \xf1\xa01.04',
+      b'\xf1\x00LX2_ SCC FHCUP 1.00 1.04 99110-S8100 \xf1\xa01.04',
```

Comment:
I'm seeing the first version instead of this new one in the latest route. The only difference is whitespace, seems like a mistake.